content/docs/authorization/deployment/_index.md

```diff
@@ -170,59 +170,93 @@ After creating the role bindings, the next logical step is to generate the acces
-    # 3) You can mix behaviors
-    #    sudo curl -k https://${AuthorizationHost}/install?namespace=powermax&proxy-port=powermax:900001&namespace=vxflexos | sh
+    kubectl apply -f /tmp/token.yaml -n isilon
    ```
 
+2. Edit the following parameters in samples/secret/karavi-authorization-config.json file in [CSI PowerFlex](https://github.com/dell/csi-powerflex/tree/main/samples), [CSI PowerMax](https://github.com/dell/csi-powermax/tree/main/samples/secret), or [CSI PowerScale](https://github.com/dell/csi-powerscale/tree/main/samples/secret) driver and update/add connection information for one or more backend storage arrays.
+
+  | Parameter | Description | Required | Default |
+   | --------- | ----------- | -------- |-------- |
+   | username | Username for connecting to the backend storage array. This parameter is ignored. | No | - |
+   | password | Password for connecting to to the backend storage array. This parameter is ignored. | No | - |
+   | intendedEndpoint | HTTPS REST API endpoint of the backend storage array. | Yes | - |
+   | endpoint | HTTPS localhost endpoint that the authorization sidecar will listen on. | Yes | https://localhost:9400 |
+   | systemID | System ID of the backend storage array. | Yes | " " |
+   | insecure | A boolean that enables/disables certificate validation of the backend storage array. This parameter is not used. | No | true |
+   | isDefault | A boolean that indicates if the array is the default array. This parameter is not used. | No | default value from values.yaml |
+
+
+Create the karavi-authorization-config secret using the following command:
+
+`kubectl -n [CSI_DRIVER_NAMESPACE] create secret generic karavi-authorization-config --from-file=config=samples/secret/karavi-authorization-config.json -o yaml --dry-run=client | kubectl apply -f -`
+
+>__Note__:  
+> - Create the driver secret as you would normally except update/add the connection information for communicating with the sidecar instead of the backend storage array and scrub the username and password
+> - For PowerScale, the *systemID* will be the *clusterName* of the array. 
+>   - The *isilon-creds* secret has a *mountEndpoint* parameter which should not be updated by the user. This parameter is updated and used when the driver has been injected with [CSM-Authorization](https://github.com/dell/karavi-authorization).
+
+3. Create the proxy-server-root-certificate secret.
+
+    If running in *insecure* mode, create the secret with empty data:
+
+      `kubectl -n [CSI_DRIVER_NAMESPACE] create secret generic proxy-server-root-certificate --from-literal=rootCertificate.pem= -o yaml --dry-run=client | k apply -f -`
+
+    Otherwise, create the proxy-server-root-certificate secret with the appropriate file:
+
+      `kubectl -n [CSI_DRIVER_NAMESPACE] create secret generic proxy-server-root-certificate --from-file=rootCertificate.pem=/path/to/rootCA -o yaml --dry-run=client | k apply -f -`
+
+#### PowerFlex
```

Comment:
Do we also need a section here for PowerMax??

Reply:
No additional configurations are needed for PowerMax, so we left out the section. I can add a note about that

Reply:
fixed